esmvaltool/cmorizers/obs/cmorize_obs_era_interim.py

```diff
@@ -0,0 +1,329 @@
+        for item in files_lst:
+            key = str(item.stem).split('_')[-1]
+            files_dict[key].append(item)
+    # Check if files are complete
+    for key in files_dict.copy():
+        if len(files_dict[key]) != len(var['files']):
+            logger.info("CMORizing %s at time '%s' needs '%s' input file/s",
+                        var['short_name'], key, len(var['files']))
+            files_dict.pop(key)
+    return files_dict
+
+
+def _extract_variable(in_file, var, cfg, out_dir):
+    if 'files' in var:
+        logger.info("CMORizing variable '%s' from input file/s '%s'",
+                    var['short_name'], [str(item) for item in in_file])
+    else:
+        logger.info("CMORizing variable '%s' from input file/s '%s'",
+                    var['short_name'], in_file)
+    attributes = deepcopy(cfg['attributes'])
+    attributes['mip'] = var['mip']
+    cmor_table = CMOR_TABLES[attributes['project_id']]
+    definition = cmor_table.get_variable(var['mip'], var['short_name'])
+
+    with catch_warnings():
+        filterwarnings(
+            action='ignore',
+            message="Ignoring netCDF variable 'tcc' invalid units '(0 - 1)'",
+            category=UserWarning,
+            module='iris',
+        )
+        filterwarnings(
+            action='ignore',
+            message="Ignoring netCDF variable 'lsm' invalid units '(0 - 1)'",
+            category=UserWarning,
+            module='iris',
+        )
+        filterwarnings(
+            action='ignore',
+            message=("Ignoring netCDF variable 'e' invalid units "
+                     "'m of water equivalent'"),
+            category=UserWarning,
+            module='iris',
+        )
+        if 'files' in var:
+            if var['operator'] == 'sum':
+                # Multiple variables case using sum operation
+                for i, item in enumerate(in_file):
+                    in_cube = iris.load_cube(
+                        str(item),
+                        constraint=utils.var_name_constraint(var['raw'][i]),
+                    )
+                    if i == 0:
+                        cube = in_cube
+                    else:
+                        cube += in_cube
+                cube.var_name = var['short_name']
+        else:
+            cube = iris.load_cube(
+                str(in_file),
+                constraint=utils.var_name_constraint(var['raw']),
+            )
+
+    _set_global_attributes(cube, attributes, definition)
+
+    # Set correct names
+    cube.var_name = definition.short_name
+    cube.standard_name = definition.standard_name
+    cube.long_name = definition.long_name
+
+    # Fix data type
+    cube.data = cube.core_data().astype('float32')
+
+    _fix_coordinates(cube, definition)
+    cube = _fix_frequency(cube, var)
+
+    # Convert units if required
+    cube.convert_units(definition.units)
+
+    # Make latitude increasing
+    cube = cube[:, ::-1, ...]
```

Comment:
Do this before fixing the coordinates, see https://github.com/ESMValGroup/ESMValTool/issues/1354#issuecomment-537930813

Reply:
In fact, it would be better to move this to the `_fix_coordinates` function as it fixes coordinates. Make sure it goes before guessing the bounds.